### DISCUSSION TITLE
Support parsing LLM model for ChatOllama

### DISCUSSION THREAD

**Discussion by kleebaum** [langfuse/langfuse#1376] [Ideas]
### Describe the feature or potential improvement

Thanks for the great product!

Currently, the model name is not recorded for ChatOllama models. Deserialization seems to be "not_implemented" yet. 
The error message is: Langfuse was not able to parse the LLM model. The LLM call will be recorded without model name.

Could you please **support parsing the LLM model for ChatOllama or Ollama**?

![grafik](https://github.com/langfuse/langfuse/assets/5443645/5e91f4a4-5407-43e1-8fa3-e248a66c477e)

Thank you!

**Comment by marcklingen** [comment]
Thanks for raising this, we'll look into a more generic solution to this. Open for PRs on this in the meantime
